src/profile/components/multiItemEditor/MultiItemEditor.tsx

```diff
@@ -51,6 +53,7 @@ function MultiItemEditor({ dataType }: Props): React.ReactElement | null {
     clearMessage,
   } = useNotificationContent();
 
+  const { showModal, modalProps } = useConfirmationModal();
```

Comment:
Feels like with hooks the order of these is often: state, changeStateFn. Not sure if it's a big deal though